ServiceApp/src/org/droidplanner/services/android/api/DroneApiUtils.java

```diff
@@ -670,10 +670,10 @@ static float generateDronie(Drone drone) {
         return (float) drone.getMission().makeAndUploadDronie();
     }
 
-    static void arm(Drone drone, boolean arm) {
+    static void arm(Drone drone, boolean arm, boolean emergencyDisarm) {
```

Comment:
@squilter the `emergencyDisarm` argument should be passed to the `MavLinkArm.sendArm()` call below, instead of hardcoding the last parameter to `false`.